connection.ts

```diff
@@ -62,15 +64,22 @@ interface Command {
   returnUint8Arrays?: boolean;
 }
 
+function isUnixConnectOptionsLike(x: object): x is UnixConnectOptionsLike {
```

Comment:
Since `RedisConnectionOptions` is passed to this function, it seems better not to check the `transport` property.
```suggestion
function isRedisConnectionOptionsUnix(x: RedisConnectionOptions): x is RedisConnectionOptionsUnix {
  return (x as RedisConnectionOptionsUnix).path != null;
}
```